src/connection.c

```diff
@@ -198,3 +198,11 @@ sds getListensInfoString(sds info) {
 
     return info;
 }
+
+sds connGetPeerUsername(connection *conn) {
+    if (conn->type && conn->type->get_peer_username) {
+        return conn->type->get_peer_username(conn);
+    }
+    return NULL;
+}
```

Comment:
We can make it `static inline` and put this together with `connGetPeerCert` in connection.h

Static inline or not doesn't matter much in practice because we compile with -O3 and link-time-optimizations so it can be inlined either way, but we can follow the existing style for similar functions.

Reply:
Done updated